### PR TITLE
Add fig support

### PR DESCRIPTION
**Pull request by denderello**
This will add fig support for local setups
### Links
- giantswarm/lockd#1

### CODE REVIEW

**Comment by denderello** [comment]
ping @ZeissS, @erkules, @zyndiecate, @teemow

File: fig.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+galera1:
+  build: .
+  command: --wsrep-cluster-address=gcomm://
```
Comment (xh3b4sd) [inline fig.yml:3]: Does this cause `galera1` to be leader?

Comment (denderello) [inline fig.yml:3]: Yes, exactly. Galera needs one peer to be started as the instance that initializes the cluster. After other peers joined all of them are equal.

Comment (xh3b4sd) [inline fig.yml:3]: But what is the trick behind? Does it not need a cluster address because communication happens just from leader to follower?

Comment (denderello) [inline fig.yml:3]: The trick here is that the peers tell each other their ip addresses and there is no concept of a leader and followers. The first peer in the cluster only opens the "session" and the others need one address of a peer to join it. As soon as others have joined the cluster the peers distribute information among them automatically.

Comment (xh3b4sd) [inline fig.yml:3]: Why does the first node, in this case the lead, not need an address?

**Comment by xh3b4sd** [comment]
LGTM

File: fig.yml (hunk 1 above)
Comment (erkules) [inline fig.yml:3]: The option --wsrep-cluster-address just tells where to find the cluster aka meetingmembers.
Being the first one starting the meeting you don't join the meeting. Thats why there is no address to join.

**Pull request merged**